README.md

```diff
@@ -1,2 +1,19 @@
-# openmrs-module-bahmnievents
-OpenMRS module to generate and publish event to the configured JMS
+OpenMRS Module Appointments Backend
```

Comment:
How do consumers know where ActiveMQ is running? Can you mention the path/name of configuration file to refer?

Reply:
Also, can you give a reference to a sample EVENT type, so people know what event to expect? (maybe in a test file)?

Reply:
Added the service name and sample payload for patient